### DISCUSSION TITLE
0.65.1

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#8678] [General]
* Improvements to localhost relay to have better performance and accurately report bind failures.
* Use /dev/ptp0 to keep guest chock in sync with the host
* Update Linux kernel to 5.15.57.1
    * Fix 9p filesystem regressions since the last v5.10 WSL2 kernel
    * Enable support for the Precision Time Protocol (PTP) clock device
    * Enable Retbleed mitigations in x86_64 builds
    * Enable nftables and traffic control
    * Enable VGEM driver
* Update Microsoft.WSLg to version 1.0.41
    * WSLg: add default x11 bell sound
    * WSLg: Update /etc/wsl.conf to set the default user to wslg
    * WSLGd: add option to start weston under gdbserver
    * WSLGd: simplify weston command line construction
    * compositor: add wslgd-notify
    * compositor: Stop trapping SIGINT
    * compositor: load xwayland module last
    * rdp: resize margin adjustment for MoveWindow/SnapArrange PDU
    * rdpaudio: use pthread_cancel instead of pthread_kill
    * xwayland: give Xwayland its own session
    * xwayland: Don't track focus for override redirect windows

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/0.65.1'>0.65.1</a>.</em>

**Comment by seanralexander** [comment]
Guest Chock?

**Reply by mwidmann** [reply]
I have issues since updating: https://github.com/microsoft/WSL/issues/8679

**Comment by YohanSciubukgian** [comment]
The changelog of the last Linux WSL version [linux-msft-wsl-5.15.57.1](https://github.com/microsoft/WSL2-Linux-Kernel/releases/tag/linux-msft-wsl-5.15.57.1) precise that Ceph will be disabled :
`Disable support for the Ceph Distributed File System`.
Does it mean that projects that rely on Ceph will not be able to run anymore using the WSL v0.65.1+ ?

**Reply by tyhicks** [reply]
That's correct. Do you rely on CEPH within WSL environments?

**Reply by NotTheDr01ds** [reply]
@tyhicks It wouldn't be the first time I've seen someone use it with WSL.  [This Stack Overflow question](https://stackoverflow.com/q/66427084/11810933) had someone doing it as well.  Not sure what the actual use-case is.
